### DISCUSSION TITLE
Any plans for French TN?

### DISCUSSION THREAD

**Discussion by eeishaan** [NVIDIA-NeMo/NeMo#3635] [Q&A] [answered]
Hi

I see that Spanish TN is being added as part of #3632 or #3555 . Is there any plan to add French TN?

Thank you

**Comment by titu1994** [comment, marked as answer]
@yzhang123 I thought french tn / itn was already added ?

**Reply by yzhang123** [reply]
we only have French ITN for now. Spanish ITN is already there and Spanish TN is on the way.

**Reply by eeishaan** [reply]
Yes, I'm excited for Spanish TN update. Do you have some idea about plans for French TN?

**Reply by yzhang123** [reply]
no we can't say anything for now, since we lack the bandwidth and linguistic knowledge. Would you be interested to add an open source contribution? I can help you with framework questions.

**Reply by eeishaan** [reply]
Thank you for your answer. I'm afraid I too don't have the bandwidth and linguistic knowledge to add french support.
